app/src/components/TimeLine.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react";
+
+function TimeLine({ isAuthenticated, gcal, beginTime, endTime }) {
+  var startTime = new Date();
+  var endTime = new Date().setHours(startTime.getHours() + 12);
+  const [currentTime, setCurrentTime] = useState(
+    new Date().toLocaleTimeString(),
+  );
+
+  //setInterval(tick({ setCurrentTime }), 1000);
+
+  var queryParams = {
+    timeMin: startTime,
+    timeMax: endTime,
+    showDeleted: true,
+    orderBy: "startTime",
+  };
+
+  useEffect(() => {}, []);
```

Comment:
Empty hook call; you can probably remove this.

Reply:
Will do.